plugins/tekton/src/utils/pipeline-filter-reducer.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-identical-title */
```

Comment:
why you needed this? 

https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-identical-title.md should we avoid this?

Reply:
There were 2 test cases with the same title. I fixed the titles but forgot to remove this.